static/js/compose.js

```diff
@@ -179,6 +179,15 @@ function create_message_object() {
         message.reply_to = recipient;
         message.private_message_recipient = recipient;
         message.to_user_ids = people.email_list_to_user_ids_string(emails);
+
+        if (page_params.realm_is_zephyr_mirror_realm === false) {
+            // split the user_ids into array i.e. "1,2" -> ["1", "2"]
+            // then turn them into number
+            message.to = _.map(message.to_user_ids.split(','), function (id) {
+                return +id;
```

Comment:
Is there a more idiomatic way to do this?

Reply:
Yes, there are two more ways to do it, `parseInt(id, 10)` and `Number(id)`. I think I will go ahead with the Number one. The reason I did this way is because I see this more often than using other methods.